CHANGELOG.rst

```diff
@@ -7,6 +7,40 @@ The format is based on `Keep a Changelog`_,
 and this project adheres to `Semantic Versioning`_.
 
 
+Version 1.2.0 (2023-02-13)
+--------------------------
+
+* Added: ``normalize`` argument to ``audiofile.convert_to_wav()``
+* Added: ``bit_depth`` argument to ``audiofile.convert_to_wav()``
+* Added: ``overwrite`` argument to ``audiofile.convert_to_wav()``
+* Added: examples to the API documentation
+* Added: example how to use ``audresample.resample()``
+  and ``audresample.remix()``
+  in the API documentation of ``audiofile.read()``
+* Added: support units and samples
+  and most other formats supported by ``audmath.duration_in_seconds()``
+  for the ``duration`` and ``offset`` arguments
+  of ``audiofile.read()``
+  and ``audiofile.convert_to_wav()``
+* Added: support for negative values
+  for the ``duration`` and ``offset`` arguments
+  of ``audiofile.read()``
+  and ``audiofile.convert_to_wav()``
+* Changed: allow ``audiofile.convert_to_wav()``
+  to be provided by only the input file.
+  The resulting WAV file
+  is then stored under the same path
+  but with ``'wav'`` as file extension
+* Fixed: ensure ``audiofile.convert_to_wav()``
+  creates identical result
+  as calling ``audiofile.read()``
+  and ``audiofile.write()``
+  independently
+* Fixed: support for non SND-files
```

Comment:
what are "SND-files"?

Reply:
Files supported by `libsndfile`. But you are right, this is not nice for a changelog. What I mean here is: all files besides WAV, FLAC, OGG

Reply:
Ok, we should be more verbose here

Reply:
I changed to

```
* Fixed: support for non WAV, FLAC, OGG files
  like MKA or MP4 
  on Windows network shares
```